R/shiny.R

```diff
@@ -1319,6 +1319,10 @@ ShinySession <- R6Class(
 
       # If we don't already have width for this output info, see if it's
       # present, and if so, add it.
+
+      # Note that all the following clientData values (which are reactiveValues)
+      # are wrapped in reactive() so that users can take a dependency on particular
+      # output info (i.e., just depend on width/height, or just depend on bg, fg, etc)
```

Comment:
Can you add: "If this function just returned a list of values from `self$clientData`, than anything that calls this function would take a reactive dependency on all of these values."

Reply:
How about:

> To put it another way, if `getCurrentOutputInfo()` simply returned a list of values from self$clientData, than anything that calls `getCurrentOutputInfo()`would take a reactive dependency on all of these values.